lms/static/js/spec/main.js

```diff
@@ -181,22 +181,7 @@
             },
             'underscore': {
                 deps: ['underscore.string'],
-                exports: '_',
-                init: function(UnderscoreString) {
```

Comment:
I don't wish to do it as part of this work. I will create a techdebt story for ECOM, as they are the one using this paradigm.